docs/en-US/component/select-v2.md

```diff
@@ -131,19 +131,39 @@ select-v2/remote-search
 
 :::
 
-## use value-key
+## Use value-key attribute
 
-:::demo when `options.value` is an object, you should set a unique identity key name for value
+when `options.value` is an object, you should set a unique identity key name for value
+
+::: tip
+
+Before ^(2.4.0), `value-key` was used both as the unique value of the selected object and as an alias for the value in `options`. Now `value-key` is only used as the unique value of the selected object, and the alias for the value in options is `props.value`.
```

Comment:
```suggestion
Before ^(2.4.1), `value-key` was used both as the unique value of the selected object and as an alias for the value in `options`. Now `value-key` is only used as the unique value of the selected object, and the alias for the value in options is `props.value`.
```

Reply:
This is a supplement to the previous #13263

Reply:
Sorry, I misunderstood.